src/webextension/list/manage/components/home-breadcrumbs.js

```diff
@@ -0,0 +1,41 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { Localized } from "fluent-react";
+import PropTypes from "prop-types";
+import React from "react";
+
+import Breadcrumbs, { Crumb } from "../../../widgets/breadcrumbs";
+import { NEW_ITEM_ID } from "../../common";
+
+export default function HomeBreadcrumbs({itemId, itemTitle, onClickHome}) {
+  if (itemId) {
+    const trimmedTitle = itemTitle.trim();
+    const titleId = `breadcrumbs-item-${itemId === NEW_ITEM_ID ? "new" : ""}`;
```

Comment:
This results in a trailing '-', and causing bad display.  I think this ought to be:
```
`breadcrumbs-item${itemId === NEW_ITEM_ID ? "-new" : ""}`
```

As-is, this renders as:

<img width="336" alt="image" src="https://user-images.githubusercontent.com/757401/38266665-7f089d74-3736-11e8-9717-678f7d44e63f.png">